src/main/java/com/diamondmarket/users/api/UserApiController.java

```diff
@@ -0,0 +1,212 @@
+		} else {
+			context.setApplicationLabel("demo");
+		}
+    	try {
+    		response = successResponse(context, userService.createUser(body), HttpStatus.CREATED);
+    		
+		} catch (Exception e) {
+			// TODO: handle exception
+			response = errorResponse(context, e, HttpStatus.BAD_REQUEST);
+		}
+    	
+    	
+        return response;
+    }
+
+    public ResponseEntity<Response> deleteUser(@RequestHeader HttpHeaders httpHeaders, @ApiParam(value = "The user that needs to be deleted",required=true ) @PathVariable("userId") String userId) {
+        // do some magic!
+       
+       	ResponseEntity<Response> response = null;
+ 		TransactionContext context = new TransactionContext();
+		if(httpHeaders.get("correlationId") != null) {
+			context.setCorrelationId(httpHeaders.get("correlationId").toString());	
+		} else {
+			context.setCorrelationId("demo");
+		}
+		if(httpHeaders.get("ApplicationLabel") != null) {
+			context.setApplicationLabel(httpHeaders.get("ApplicationLabel").toString());
+		} else {
+			context.setApplicationLabel("demo");
+		}
+		
+    	try {
+    		String deleteUser = userService.deleteUser(userId);
+    		
+    		response = successResponse(context,deleteUser, HttpStatus.OK);
+    		
+		} catch (Exception e) {
+			// TODO: handle exception
+			response = errorResponse(context, e, HttpStatus.BAD_REQUEST);
+		}
+    	
+    	
+        return response;
+    }
+
+    public ResponseEntity<Response> getUserById(@RequestHeader HttpHeaders httpHeaders, @ApiParam(value = "",required=true ) @PathVariable("userId") String userId) {
+        // do some magic!
+       	ResponseEntity<Response> response = null;
+ 		TransactionContext context = new TransactionContext();
+		if(httpHeaders.get("correlationId") != null) {
+			context.setCorrelationId(httpHeaders.get("correlationId").toString());	
+		} else {
+			context.setCorrelationId("demo");
+		}
+		if(httpHeaders.get("ApplicationLabel") != null) {
+			context.setApplicationLabel(httpHeaders.get("ApplicationLabel").toString());
+		} else {
+			context.setApplicationLabel("demo");
+		}
+        	try {
+        		
+        		User user = userService.getUser(userId);
+        		
+        		response = successResponse(context, user, HttpStatus.OK);
+        		
+    		} catch (Exception e) {
+    			// TODO: handle exception
+    			response = errorResponse(context, e, HttpStatus.BAD_REQUEST);
+    		}
+        	
+        	
+            return response;
+    	
+    }
+
+    public ResponseEntity<Response> loginUser(@RequestHeader HttpHeaders httpHeaders, @NotNull@ApiParam(value = "The user name for login", required = true) @RequestParam(value = "username", required = true) String username,
+         @NotNull@ApiParam(value = "The password for login in clear text", required = true) @RequestParam(value = "password", required = true) String password) {
+        // do some magic!
+       	ResponseEntity<Response> response = null;
+ 		TransactionContext context = new TransactionContext();
```

Comment:
@shahmitesh51  & @Vishalpopat  Please follow below coding Standards.
1. exclude "// TO_DO Auto generated"
2. Indentation should not be more than one tab
3. Remove redundant code by including one method.
4. Remove unused imports.